Calculator/Calculator.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Looks like you didn't include "calculator library" into your repository. Please create a folder with the name chernoAlieu.Calculator and include both projects inside of it. Let me know if you need help. 

![image](https://user-images.githubusercontent.com/108711476/216179596-4f4d4cd1-42dd-444e-b79c-e053c4eb9753.png)

Reply:
Should I do that within Visual Studio or from file explorer?

Reply:
Whenever you're dealing with folders I think it's easier to use file explorer

Reply:
Alright, thank you. Right away then